gpsd_client/src/client.cpp

```diff
@@ -1,10 +1,33 @@
+#include <libgpsmm.h>
+
+// gps.h defines some macros which conflict with enum values in NavSatStatus.h,
+// so we map them to new names before including other headers.
+#ifdef STATUS_FIX
+  enum {
+    GPSD_STATUS_NO_FIX   = STATUS_NO_FIX,
+    GPSD_STATUS_FIX      = STATUS_FIX,
+    GPSD_STATUS_DGPS_FIX = STATUS_DGPS_FIX,
+  };
+  #undef STATUS_NO_FIX
+  #undef STATUS_FIX
+  #undef STATUS_DGPS_FIX
+#else
+  // Renamed in gpsd >= 3.23.1 (commits d4a4d8d3, af3b7dc0, 7d7b889f) without
+  // revising the API version number.
+  enum {
+    GPSD_STATUS_NO_FIX   = STATUS_UNK,
+    GPSD_STATUS_FIX      = STATUS_GPS,
+    GPSD_STATUS_DGPS_FIX = STATUS_DGPS,
+  };
+#endif
+
```

Comment:
I think this is obsolete with some of the other changes that have been made to `ros2-devel`